rsconf/package_data/mongod/mongod.conf.jinja

```diff
@@ -0,0 +1,19 @@
+# for documentation of all options, see:
+#   http://docs.mongodb.org/manual/reference/configuration-options/
+net:
+  port: {{ mongod.port }}
+  bindIp: {{ mongod.ip }}
+processManagement:
+  fork: true
+  pidFilePath: /var/run/mongodb/mongod.pid
+  timeZoneInfo: /usr/share/zoneinfo
+security:
+  authorization: enabled
+storage:
+  dbPath: {{ mongod.data_d }}
+  journal:
+    enabled: true
+systemLog:
+  destination: file
+  logAppend: true
+  path: /var/log/mongodb/mongod.log
```

Comment:
Does mongod come with a logrotate? If not, we should build our own and put the logs in /srv/mongod.

Reply:
It doesn't. Added one 6c04a19

Reply:
They do. That code is outdated after  https://github.com/radiasoft/rsconf/commit/6c04a196e62e07639a1c84a428ab1cd8fdfebe21

See
https://github.com/radiasoft/rsconf/commit/6c04a196e62e07639a1c84a428ab1cd8fdfebe21#diff-a74861d485a09fe805e46df8af0f6b164c63eddb08c2685b38593e0204cd10acR19